src/pwa/lib/featureFlags.ts

```diff
@@ -0,0 +1,6 @@
+export const featureFlags = () => {
+    return {
+        isEnableNotification: true,
+        isEnableWebAuth: true
```

Comment:
Not all features are covered. I would include and utilize following:
- isEnableCalling
- isEnableCamera
- isEnableLocationServices

Reply:
I updated. when feature is disable, the page with be redirected to disableFeature page

Reply:
I would still like to have isEnableNotification and isEnableWebAuth in feature flags.
If isEnableNotification is false, remove component from settings, and try to exclude all Firebase integration.
if isEnableWebAuth is false, remove component from settings, and try to exclude all Web Authentication API integration.

Reply:
I bring it back. Please help me take a look at the last commit